flexmeasures_client.py

```diff
@@ -127,9 +127,23 @@ def trigger_schedule(self, *args, **fnc_kwargs):
             target = self.args["fm_car_max_soc_in_kwh"]
             target_datetime = (time_round(datetime.now(tz=pytz.utc), resolution) + timedelta(days=7)).isoformat()
         else:
-            target = search_for_kwh_target(car_reservation["attributes"]["description"])
-            if target is None:
-                target = self.args["fm_car_max_soc_in_kwh"]
+            # Depending on the type of calendar the description or message contains the possible target.
+            text_to_search_in = car_reservation["attributes"]["message"] + " " + car_reservation["attributes"]["description"]
+
+            target_in_procent = search_for_kwh_target(text_to_search_in)
+            if target_in_procent is None:
+                target = self.CAR_MAX_SOC_IN_KWH
+            else:
+                if target_in_procent > 100:
+                    # Prevent accidental input larger than 100%
+                    target_in_procent = 100
+                elif target_in_procent < 30:
```

Comment:
Shouldn't this value come from a user setting?

Reply:
I'm not sure if there is much value in such settings.
These checks are mainly here to prevent the user from making mistakes. The user already choses by setting the value in the calendar.
And 100% is a "natural limit" that we also use if no value is set.
The 30 seems a logical value, just above our minimum SoC. Maybe first make this minimum SoC (for max charge and schedule) a setting.